src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  document.body.append(message);
+
+  message.classList.add('notification', type);
+
+  const messageTitle = document.createElement('h2');
+
+  messageTitle.className = 'title';
```

Comment:
Consider using `classList.add()` instead of `className` for better readability and maintainability. 📝